7/Dockerfile.fedora

```diff
@@ -29,6 +30,7 @@ RUN INSTALL_PKGS="gettext hostname nss_wrapper bind-utils varnish" && \
     rpm -V $INSTALL_PKGS && \
     fix-permissions $VARNISH_CONFIGURATION_PATH && \
     fix-permissions /var/lib/varnish && \
+    varnishd -V 2>&1 | grep -qe "varnish-7\." && echo "Found VERSION 7" && \
```

Comment:
It would be nice if we use newly introduced VARNISH_VERSION variable here, instead of hardcoded varnish-7.

Reply:
But if it would be problem to do it with distgen I am OK with this solution too. However, then we do not really need the new variable, do we?

Reply:
The files are dist-gen generated. But you are right. I will do it.

Reply:
Fixed by https://github.com/sclorg/varnish-container/pull/132/commits/e3d43d5a82ec1e47eda5eaf15fa122feeb5ec5e6